server/wavefront.go

```diff
@@ -58,6 +59,41 @@ func (wf *WaveFrontProvider) init() error {
 func (wf *WaveFrontProvider) getType() string {
 	return WAVEFRONT_TYPE
 }
+
+// This one is part of alpha, needs to be tested
```

Comment:
Please provide better explanation to what exactly is alpha

Reply:
Added proper comments describing about the alpha.